src/Browscap/Data/PropertyHolder.php

```diff
@@ -270,6 +270,48 @@ public function isOutputProperty(string $propertyName, WriterInterface $writer)
     }
 
     /**
+     * Determine if the specified property is an "extra" property (that should
+     * be included in any version of the files)
+     *
+     * @param string          $propertyName
+     * @param WriterInterface $writer
+     *
+     * @return bool
+     */
+    public function isDeprecatedProperty(string $propertyName, WriterInterface $writer) : bool
+    {
+        $outputProperties = [
+            'Win16' => 1,
+            'Win32' => 1,
+            'Win64' => 1,
+            'isMobileDevice' => 1,
+            'isTablet' => 1,
+            'Crawler' => 1,
+            'AolVersion' => 1,
+        ];
+
+        if (isset($outputProperties[$propertyName])) {
+            return true;
+        }
+
+        if (in_array($writer->getType(), [WriterInterface::TYPE_CSV, WriterInterface::TYPE_XML])) {
```

Comment:
Should this be here? Seems like leftover from copy/paste.

These aren't deprecated properties are they?